plugins/private_message/include/golos/plugins/private_message/private_message_operations.hpp

```diff
@@ -9,16 +9,39 @@ namespace golos { namespace plugins { namespace private_message {
     struct private_message_operation: public base_operation {
         account_name_type from;
         account_name_type to;
-        uint64_t nonce = 0; /// used as seed to secret generation
+        uint64_t nonce;
         public_key_type from_memo_key;
         public_key_type to_memo_key;
-        uint32_t checksum = 0;
+        uint32_t checksum;
+        bool update = false;
         std::vector<char> encrypted_message;
 
         void validate() const;
         void get_required_posting_authorities(flat_set<account_name_type>& a) const;
     };
 
+    struct private_delete_message_operation: public base_operation {
+        account_name_type from;
+        account_name_type to;
+        uint64_t nonce = 0;
+        time_point_sec from_date;
+        time_point_sec to_date;
```

Comment:
The names `from_date`, `to_date` can be misunderstood as related with `from` and `to` fields. Maybe it will be clearer to rename them to `start_time`/`end_time` or something like that.
It's not a big issue, but they already confused me for a moment.

Reply:
Renamed to start_date and stop_date.